app/models.py

```diff
@@ -0,0 +1,230 @@
+from peewee import *
+
+DATABASE_NAME = "database.db"
+database = SqliteDatabase(DATABASE_NAME)
+
+
+# Database models
+class BaseModel(Model):
```

Comment:
Are you referring to each model's attribute validation? Because if so, it is possible (and also nicer, in my opinion) to use SQL directives for that (which peewee supports)

Reply:
Yep, if this is possible to configure in the DB itself (using peewee) that would be best!